scripts/natural_language_inference/decomposable_atten.py

```diff
@@ -0,0 +1,100 @@
+# pylint: disable=C0103,E1101,R0914
+"""
+decomposable_atten.py
+
+Part of NLI script of gluon-nlp. Implementation of Decomposable Attention.
+Copyright 2018 Mengxiao Lin <linmx0130@gmail.com>.
+"""
+
+from mxnet import nd, gluon
+from mxnet.gluon import nn
+
+class IntraSentenceAtten(gluon.Block):
+    """
+    Intra Sentence Attantion block.
+    """
+    def __init__(self, inp_size: int, hidden_size: int, max_length=10, **kwargs):
+        super(IntraSentenceAtten, self).__init__(**kwargs)
+        self.max_length = max_length
+        self.hidden_size = hidden_size
+        with self.name_scope():
+            self.func = nn.Sequential()
+            self.func.add(nn.Dense(hidden_size, in_units=inp_size, activation='relu'))
+            self.func.add(nn.Dense(hidden_size, in_units=hidden_size, activation='relu'))
+            self.func.add(nn.Dense(hidden_size, in_units=hidden_size))
+
+    def forward(self, *args):
```

Comment:
Why not 
```suggestion
    def forward(self, feature_a, *args):
```
rather than using args[0]?